qulice-pmd/src/test/java/com/qulice/pmd/PMDValidatorTest.java

```diff
@@ -491,4 +497,19 @@ public void allowsAssertFail()
         ).validate();
     }
 
+    /**
+     * PMDValidator can allow non-static, non-transient fields.
+     * @throws Exception If something wrong happens inside.
+     */
+    @Test
+    public void allowNonTransientFields() throws Exception {
```

Comment:
@vkuchyn The answer is here http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names :)
> It's important to start the first sentence of your JavaDoc with the name of the class you're testing followed by can. So, your first sentence should always be similar to "somebody can do something".

> The method name will state exactly the same, but without the subject. If I add a subject at the beginning of the method name, I should get a complete English sentence, as in above example: "HttpRequest returns its content in Unicode".